addons/event/models/event_question.py

```diff
@@ -19,19 +20,21 @@ class EventQuestion(models.Model):
         ('phone', 'Phone'),
         ('company_name', 'Company'),
     ], default='simple_choice', string="Question Type", required=True)
-    event_type_id = fields.Many2one('event.type', 'Event Type', ondelete='cascade', index='btree_not_null')
-    event_id = fields.Many2one('event.event', 'Event', ondelete='cascade', index='btree_not_null')
+    event_type_ids = fields.Many2many('event.type', string='Event Type', index='btree_not_null')
+    event_ids = fields.Many2many('event.event', string='Event', index='btree_not_null')
```

Comment:
Indexes should be removed, they have no usage anymore since we go through an intermediate table